contract/test/TipsProxyt.spec.ts

```diff
@@ -2,7 +2,7 @@ import { ethers, ignition } from 'hardhat'
 import { expect } from 'chai'
 
 import TipsModule from '../ignition/modules/ProxyModule'
-describe.only('Tips Proxy', function () {
+describe('Tips Proxy', function () {
   describe('Proxy interaction', async function () {
```

Comment:
had to remove .only because it doesn't run the other tests

Reply:
LGTM